test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -232,8 +232,7 @@ private void ValidateDump(int expectedDumpCount = 1)
             }
 
             var dumps = attachments
-                .Where(a => a.EndsWith(".dmp"))
```

Comment:
Yeah this makes the code less readable, even if it is merged with a different condition.

Reply:
i will revert if you like

Reply:
reverted